### PR TITLE
Improve landing page UI

### PR DESCRIPTION
**Pull request by anqitu**

### CODE REVIEW

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -93,48 +93,44 @@ <h3 class="display-5">PhotoBook is the website for photography enthusiasts.</h3>
...
+        <div class="mx-md-3 pt-4 text-center overflow-hidden description-card">
+          <div class="py-3">
             <img class="landing-icon" alt="camera icon" src="images/home-page-icon/camera.png" />
-            <p class="text-justify px-3 font-weight-lighter">
+            <h5>Share Your Photographs</h5>
+            <p class="text-center px-3 font-weight-lighter description-card-detail">
               Sharing your photographs has never been easier! Upload photographs from
-              <a href="/user-page.html">your home page</a>
-              with a click.
+              <a href="/user-page.html">your home page</a> with a click. PhotoBook will then categorize your images and detects landmarks automatically.
```
Comment (amad-person) [inline src/main/webapp/index.html]:
```suggestion
              <a href="/user-page.html">your home page</a> with a click. PhotoBook will then categorize your images and detect landmarks automatically. 
```

File: src/main/webapp/js/index-page-loader.js [hunk 2]
```diff
@@ -10,12 +10,15 @@ const carouselItems = [
   },
```
Comment (amad-person) [inline src/main/webapp/js/index-page-loader.js:10]: Typo in location for Singapore

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -61,11 +61,11 @@
...
       <div class="landing-tagline">
-        <h1 class="display-4 text-center">
-          <big><big>PhotoBook</big></big>
+        <h1 class="display-4 text-center font-weight-bold">
+          <big>PhotoBook</big>
         </h1>
-        <h3 class="font-weight-lightest text-center">
-          Shoot. Share. Explore.
+        <h3 class="font-weight-light text-center">
+          Shoot &nbsp & &nbsp Share
```
Comment (amad-person) [inline src/main/webapp/index.html]: Can we use the previous tagline?

**Review by amad-person — DISMISSED** [review]

File: src/main/webapp/index.html (hunk 3 above)
Comment (anqitu) [reply src/main/webapp/index.html]: Sure!

File: src/main/webapp/js/index-page-loader.js (hunk 2 above)
Comment (anqitu) [reply src/main/webapp/js/index-page-loader.js:10]: Nice catch!

**Review by AditiSaini — APPROVED** [review]

**Pull request merged**